### PR TITLE
Bento 2197 - Refactored Tab component

### PR DESCRIPTION
**Pull request by Senait7-ux**
## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

### CODE REVIEW

**Review by iksheth — CHANGES_REQUESTED** [review]

**Comment by iksheth** [comment]
@Senait7-ux Similarly like #752 Tab component is currently living in bento-core folder under bento-frontend (Bento Frontend's Reference Implementation). As we have new structure implemented. Can you migrate login component to packages folder? and import it like @bento-core/tab or @bento-core/tabs?

**Comment by Senait7-ux** [comment]
Bento 2197: https://github.com/CBIIT/bento-frontend/pull/752

**Comment by Senait7-ux** [comment]
> @Senait7-ux Similarly like #752 Tab component is currently living in bento-core folder under bento-frontend (Bento Frontend's Reference Implementation). As we have new structure implemented. Can you migrate login component to packages folder? and import it like @bento-core/tab or @bento-core/tabs?

https://github.com/CBIIT/bento-frontend/pull/752

File: packages/tab/src/index.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import Tabs from './Tabs'
```
Comment (iksheth) [inline packages/tab/src/index.js:1]: Missing ";" at the end of the line. This might fail the build while building on DEV.

**Review by iksheth — CHANGES_REQUESTED** [review]

**Review by iksheth — APPROVED** [review]

**Comment by iksheth** [comment]
Merge Conflicts.

**Comment by iksheth** [comment]
Merge Conflicts resolved.

**Pull request merged**